admin/meetings/views.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import unicode_literals
+
+from django.views.generic import ListView, FormView
+from django.views.defaults import page_not_found
+from django.core.urlresolvers import reverse
+
+from framework.auth.core import get_user
+from website.models import Conference
+from website.conferences.exceptions import ConferenceError
+
+from admin.base.utils import OSFAdmin
+from admin.meetings.forms import MeetingForm
+from admin.meetings.serializers import serialize_meeting
+
+
+class MeetingListView(OSFAdmin, ListView):
+    template_name = 'meetings/list.html'
+    paginate_by = 10
+    paginate_orphans = 1
+    context_object_name = 'meeting'
+
+    def get_queryset(self):
+        return Conference.find()
+
+    def get_context_data(self, **kwargs):
+        queryset = kwargs.pop('object_list', self.object_list)
+        page_size = self.get_paginate_by(queryset)
+        paginator, page, queryset, is_paginated = self.paginate_queryset(
+            queryset, page_size
+        )
+        kwargs.setdefault('meetings', map(serialize_meeting, queryset))
+        kwargs.setdefault('page', page)
+        return super(MeetingListView, self).get_context_data(**kwargs)
+
+
+class MeetingFormView(OSFAdmin, FormView):
+    template_name = 'meetings/detail.html'
+    form_class = MeetingForm
+
+    def get(self, request, *args, **kwargs):
+        try:
+            return super(MeetingFormView, self).get(request, *args, **kwargs)
+        except ConferenceError:
+            return handle_conference_error(request, 'meeting',
+                                           self.kwargs.get('endpoint'))
+
+    def post(self, request, *args, **kwargs):
+        try:
+            return super(MeetingFormView, self).post(request, *args, **kwargs)
+        except ConferenceError:
+            return handle_conference_error(request, 'meeting',
+                                           self.kwargs.get('endpoint'))
+
+    def get_context_data(self, **kwargs):
+        kwargs.setdefault('meeting', serialize_meeting(
+            Conference.get_by_endpoint(self.kwargs.get('endpoint'))
+        ))
+        return super(MeetingFormView, self).get_context_data(**kwargs)
+
+    def get_initial(self):
+        self.initial = serialize_meeting(
```

Comment:
I don't think you need to get and serialize the conference by endpoint both here and in `get_context_data`. Is there a reason?